phpBB/language/en/acp/attachments.php

```diff
@@ -111,6 +111,10 @@
 
 	'IMAGE_LINK_SIZE'			=> 'Image link dimensions',
 	'IMAGE_LINK_SIZE_EXPLAIN'	=> 'Display image attachment as an inline text link if image is larger than this. To disable this behaviour, set the values to 0px by 0px.',
+	'IMAGE_QUALITY'				=> 'Quality for image compression',
+	'IMAGE_QUALITY_EXPLAIN'		=> 'Higher value means lower compression/larger file size. Default 90%. Setting is only effective if max image dimensions set to value other than 0px v 0px, but will then apply to all uploaded JPEG images',
```

Comment:
I don't like this 0px V 0 px wording. Maybe something more like "0px by 0py"?

Reply:
Thanks for the pickup, yes that's a typo. If you mean it should read "0px by 0px" (like the line above) then I agree.